setup.py

```diff
@@ -56,6 +55,7 @@
         'Environment :: No Input/Output (Daemon)',
         'Topic :: Internet :: WWW/HTTP',
     ],
+    python_requires='>=3.7',
```

Comment:
I think it's okay to leave this off, since Python 3.6 does work, but we aren't making any guarantees?

Reply:
Sure, this way what will happen is that if someone try to install in a version below 3.7 it will not succeed. Removed.